Topics/Tech_Stacks/Apache_Superset.md

```diff
@@ -38,6 +42,46 @@ This message shows up due to an update in version 2.1.0 to force secure configur
 1. The command `pip install apache-superset` doesn't work. This is because Apache Superset currently supports python version 3.8 and 3.9. Any python versions that's lower or higher will result in a failure.
 
 
+## Creating a Custom Plugin:
+
+**To get started on creating a custom plugin, you can follow the instruction [HERE](https://superset.apache.org/docs/contributing/creating-viz-plugins/)**
+
+There are example plugins [example](https://github.com/preset-io/superset-plugin-chart-liquid) which you can reference. Furthermore, this [youtube tutorial](https://www.youtube.com/watch?v=LDHFY9xTzls) can help you as well.
+
+### Potential Issues Creating a Custom Plugin:
```

Comment:
Excellent details listing out the potential issue with the plug-ins as well as how to fix it. However what would happen if you add the plug-in folder into the superset folder? Will it cause an error?

In the case of an error, what does the user need to do in order to resolve it?

Questions above need to be answered before merging.

Reply:
Ah yes I forgot to mention: You can put the your-plugin folder anywhere in your machine EXCEPT in the superset/superset-frontend/plugins folder. This is because by default superset runs some processes on all folders in the plugin folder and it will cause errors for your custom plugin.

I will add this note in the md file. Thanks for your review.